README.md

```diff
@@ -32,42 +27,33 @@ func main() {
 See the `examples` folder for more examples of Trill’s language features.
 
 
-## Backends
+### LLVM
 
-Trill has two backends, one for LLVM IR, and one for JavaScript. The JavaScript backend does not accurately represent the semantics of Trill, and really only exists to provide very simple interpreting of Trill programs inside the iOS app.
-
-### LLVM IR
 The main backend for Trill is LLVM. It uses LLVM and Clang’s C APIs to call into LLVM’s code generation and clang importing features from within Swift. It currently makes a best effort to import all supported declarations from the C standard library headers as `foreign` declarations, ready for use in Trill.
 
-### iOS App
-This repo also contains an iOS app that will perform semantic syntax highlighting, error reporting, and timing information for Trill. It will also convert Trill to JavaScript and execute it live on the device.
-
-When using Trill in a JavaScript context, you can forward-declare certain types and methods that exist within the JavaScript standard library and use them from Trill as you would from JavaScript. Trill will ensure that your types are matched up, then emit the equivalent dynamic JavaScript.
+## Building and Using
 
-```swift
-foreign type Date {
-  foreign func unixTimestamp() -> Int
-  func getTime() -> Int {
-    return self.unixTimestamp() / 1_000
-  }
-}
+Trill builds on macOS and Linux using CMake. We have a convenient build script that obviates the need for using CMake directly.
+Once you've got LLVM and CMake installed, you'll need to generate pkgconfig files for LLVM and Clang -- we have a tool for this in the
+`utils` directory. You'll only need to run it once.
 
-func main() {
-  println(Date().getTime())
-}
+```bash
+swift utils/make-pkgconfig.swift
```

Comment:
this seems to be gone?

Reply:
I’m debating making that file...
I could just merge the make-pkgconfig files from LLVMSwift and ClangSwift and just put them in this repo?